types/rclone.js/rclone.js-tests.ts

```diff
@@ -0,0 +1,39 @@
+import rclone, { promises as rclonePromise } from "rclone.js";
```

Comment:
```suggestion
import rclone = require("rclone.js");
import { promises as rclonePromise } from "rclone.js";
```
or use `import * as rclone` syntax

Reply:
Addressed in latest commit